components/elm/src/main/perfMod_GPU.F90

```diff
@@ -7,25 +7,25 @@ module perfMod_GPU
 
 contains
 
-  subroutine  t_startGPU(event)
```

Comment:
So, you fallback to plain GPTL calls (t_startf) when there is no OpenACC. And it's presently just a placeholder for the GPU side timing code? What are you planning to use?

Reply:
Yes it just a placeholder to avoid having to comment out all the timing functions for cpu runs.  A future development I'm interested in would be to make calls to the CUDA  clock() device function to get rough timing for sections of more complicated kernels on a GPU block level.  

 For profiling purposes, I currently either use the built-in PGI_ACC_TIME (which is CUPTI based)  for running the full ELM code or use nvprof/nv_nsight_cu_cli to profile certain combinations of subroutines in a unit-testing environment.

Also, I have the timing data comparing this branch to master for a few runs on Summit.  The compset I used was I1850GSWCNPRDCTCBC with hcru_hcru resolution with ~500 gridcells per process.  The run length was 31 days and I just compared the model_timing_stats times.   It seems that this branch is consistently a little bit faster.  The "CNUpdate2" is nearly twice as fast. 

The times on summit seem pretty consistent and here are the Walltotal times for a set of runs.  Let me know if you want the files or a different comparison.   I include the dyn_subgrid since this branch made many changes to that section.

**Timer**            **master**         **branch**
l:elm_run            9.97e+04           9.48e+04
l:dyn_subgrid     7.86e+03           7.70e+03
l:CNUpdate2      5.46e+03           2.70e+03

Reply:
@sarats Does anything else need to be resolved concerning the timers or for performance?

Reply:
Timings look good. 

The placement of perfMod_GPU.F90 under the land model sources still feels a bit out of place. As it is, it is a stub on both CPU and GPU fronts.  I'm Ok with moving forward for now but we should probably converge on requirements if there is a need for fine-granularity of timers within GPU code.

Reply:
@peterdschwartz I approved this PR but there is still a need to converge on GPU timer requirements for the future.